dev-client/src/screens/SoilScreen/ColorScreen/ColorGuideScreen.tsx

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright © 2024 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+import {Button} from 'native-base';
+import {useCallback} from 'react';
+import {useTranslation} from 'react-i18next';
+import {Image, ScrollView} from 'react-native';
+import {BulletList} from 'terraso-mobile-client/components/BulletList';
+import {Icon} from 'terraso-mobile-client/components/Icons';
+import {
+  Box,
+  Column,
+  Paragraph,
+  Row,
+  Text,
+} from 'terraso-mobile-client/components/NativeBaseAdapters';
+import {useNavigation} from 'terraso-mobile-client/navigation/hooks/useNavigation';
+import {ScreenScaffold} from 'terraso-mobile-client/screens/ScreenScaffold';
+import {SoilPitInputScreenProps} from 'terraso-mobile-client/screens/SoilScreen/components/SoilPitInputScreenScaffold';
+
+export const ColorGuideScreen = (
+  props: SoilPitInputScreenProps | undefined,
+) => {
+  const {t} = useTranslation();
+  const navigation = useNavigation();
+
+  const onGoBack = useCallback(() => navigation.pop(), [navigation]);
+  const onTakePhoto = useCallback(
+    () =>
+      navigation.replace('COLOR_ANALYSIS', {
+        photo: 0,
+        pitProps: props,
+      }),
+    [navigation, props],
+  );
+
+  const stepContent = [
+    <>
+      <BulletList
+        data={[1, 2, 3, 4, 5]}
+        renderItem={i => (
+          <Text>{t(`soil.color.guide.step1.bullets.${i}`)}</Text>
+        )}
+      />
+      <Box
+        width="100%"
+        height="200px"
+        alignItems="center"
+        justifyContent="center"
+        backgroundColor="grey.300">
+        <Text>photo of post-its</Text>
```

Comment:
question: is this just placeholder text until we get the art?

Reply:
yes